scripts/start.py

```diff
@@ -139,6 +145,17 @@ def main(args=None):
             '--config', 'webpack.dev.config.ts', '--watch']))
         # Give webpack few seconds to do the initial compilation.
         time.sleep(10)
+        # Redis-cli is only required in a development environment.
```

Comment:
Hm sounds like we should put it in oppia_tools, then.

Reply:
Done.